src/styles/index.scss

```diff
@@ -1,3 +1,102 @@
-body {
+@import './variables';
+
+.page {
+  height: 100%;
   margin: 0;
+  display: flex;
```

Comment:
It's a good practice to reset default browser styles using a CSS reset or a modern CSS Reset library. This will help you avoid cross-browser differences in default styling.